azuredevops/internal/acceptancetests/resource_servicehook_webhook_test.go

```diff
@@ -0,0 +1,169 @@
+// +build all resource_servicehook_webhook
+// +build !exclude_servicehook
+
+package acceptancetests
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/acceptancetests/testutils"
+)
+
+func TestAccServiceHookWebhook_basic(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	eventType := "git.push"
+	url := "https://webhooks.org"
+
+	resourceType := "azuredevops_servicehook_webhook"
+	tfSvcHookNode := resourceType + ".test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, nil) },
+		Providers:    testutils.GetProviders(),
+		CheckDestroy: testutils.CheckServiceHookWebhookDestroyed(resourceType),
+		Steps: []resource.TestStep{
+			{
+				Config: hclSvcHookWebhookResourceBasic(projectName, eventType, url),
+				Check: resource.ComposeTestCheckFunc(
+					testutils.CheckServiceHookWebhookExistsWithEventTypeAndUrl(tfSvcHookNode, eventType, url),
```

Comment:
Duplicate with `resource.TestCheckResourceAttr(tfSvcHookNode, "event_type", eventType),` and `resource.TestCheckResourceAttr(tfSvcHookNode, "url", url),`